IntelFsp2Pkg/Tools/SplitFspBin.py

```diff
@@ -1,6 +1,6 @@
 ## @ FspTool.py
 #
-# Copyright (c) 2015 - 2018, Intel Corporation. All rights reserved.<BR>
+# Copyright (c) 2015 - 2019, Intel Corporation. All rights reserved.<BR>
```

Comment:
Are these changes to this file coming from EDK2?

Reply:
Yes, it is from the EDK2